compiler/optimizer/OMRSimplifierHelpers.cpp

```diff
@@ -579,6 +579,11 @@ bool decodeConversionOpcode(TR::ILOpCode op, TR::DataType nodeDataType, TR::Data
       TR::ILOpCodes opValue = op.getOpCodeValue();
       for (int i = 0; i < TR::NumAllTypes; i++)
           {
+          if(TR::NumOMRTypes == i)
+             {
+             i++; // skip it
```

Comment:
I don't think the increment is necessary (or correct for that matter).

Reply:
For opcode `bu2l` it has property `ILProp1::Conversion` but now shown in `conversionMap`, so it will lead to assertion failure. So far there're 66 opcodes has the property but a lot less shown in conversionMap, maybe there's a better solution.

Reply:
My comment wasn't about whether skipping this entry is necessary, but whether you need to do an explicit increment to achieve that.  If all you're doing is skipping the `TR::NumOMRTypes` entry then the `continue` should take care of that for you.  Unless I'm missing some subtlety here.

Reply:
Sorry, my mistake. Now updated with `continue` only to skip it. Thx!